libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -2061,7 +2061,7 @@ private static String toHtml(SpannableStringBuilder builder,
         clearMetaSpans(builder);
         parser.syncVisualNewlinesOfBlockElements(builder);
         Format.postProcessSpannedText(builder, true);
-        return EndOfBufferMarkerAdder.removeEndOfTextMarker(parser.toHtml(builder, false));
+        return EndOfBufferMarkerAdder.removeEndOfTextMarker(parser.toHtml(builder, false, false));
```

Comment:
I think we can use the `@JvmOverloads` annotation in the Aztec method in Kotlin to allow the Java side to call the method without the new extra parameter). Check out [this answer on SO](https://stackoverflow.com/a/34518024).

Reply:
Thanks for the hint. 
I have added annotation and now we don't need to pass extra param to the Kotlin method.